src/catalog/catalog.cpp

```diff
@@ -173,10 +219,9 @@ void Catalog::Bootstrap(transaction::TransactionContext *txn) {
 }
 
 // TODO(pakhtar): resolve second arg.
-void Catalog::AddColumnsToPGAttribute(transaction::TransactionContext *txn, db_oid_t db_oid,
-                                      const std::shared_ptr<storage::SqlTable> &table) {
-  Schema schema = table->GetSchema();
-  std::vector<Schema::Column> cols = schema.GetColumns();
+void Catalog::AddColumnsToPGAttribute(transaction::TransactionContext *txn, db_oid_t db_oid, SqlTableHelper *table) {
```

Comment:
I would not call this `PgAttribute`. We shouldn't leak the Postgres naming convention for tables into our API.

Reply:
I agree, my plan for Postgres-isms is to cleanly isolate them into separate files and possibly a sub-directory to make it very clear where the boundary between internal and Postgres logic is.